examples/go-daemon/daemonapi.md

```diff
@@ -1,15 +1,41 @@
 # Table of Contents
 - [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
 - [Installation](#installation)
+  - [Script (Recommended)](#script-recommended)
+  - [Manual](#manual)
 - [Usage](#usage)
   - [Example](#example)
   - [Getting Started](#getting-started)
 
 # Introduction
 This is a libp2p-backed daemon wrapping the functionalities of go-libp2p for use in Nim. <br>
 For more information about the go daemon, check out [this repository](https://github.com/libp2p/go-libp2p-daemon).
+> **Required only** for running the tests.
+
+# Prerequisites
+Go with version `1.15.15`.
+> You will *likely* be able to build `go-libp2p-daemon` with different Go versions, but **they haven't been tested**.
 
 # Installation
+Follow one of the methods below:
+
+## Script (Recommended)
+Run the build script while having the `go` command pointing to the correct Go version.
+We recommend using `1.15.15`, as previously stated.
+```sh
+./scripts/build_p2pd.sh
+```
+If everything goes correctly, the binary (`p2pd`) should be built and placed in the correct directory.
+If you find any issues, please head into our discord and ask for our asistance.
+
+After successfully building the binary, remember to add it to your path so it can be found. You can do that by running:
+```sh
+export PATH="$PATH:$HOME/go/bin"
+```
+> **Tip:** To make this change permanent, add the command above to your `.bashrc` file.
+
+## Manual
```

Comment:
Could we update the manual part as well? I'm not sure it is correct.

Reply:
Could you elaborate on what's wrong? Aside from the first block which relates to "installing nim-libp2p", that obviously doesn't belong here.
I haven't manually installed `go-libp2p-daemon`, so I don't know what's missing/wrong there.

Reply:
`git submodule update --init --recursive` seems it's saying nim-libp2p uses git submodule and I don't think it's true anymore.

`git checkout v0.0.1` I don't think that's the `go-libp2p-daemon` being used now.

Reply:
Wouldn't it be better then to just stick to the `script` based installation?

Reply:
I'm fine with removing the manual part.